frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -131,11 +133,11 @@ watch(
                 <div>
                     <InputText
                         id="forestClientInput"
-                        placeholder="Enter and verify the client ID"
+                        placeholder="Enter and verify the client number"
                         v-bind="field"
                         class="w-100 custom-height"
                         @input="cleanupForestClientNumberInput()"
-                        @keypress.enter="
+                        @keydown.enter.prevent="
```

Comment:
Replaced this because `keydown` has been deprecated.

Reply:
Hi Nick, sorry I'm a little bit confused, keydown is deprecated or keypress?

Reply:
It is `keypress`... Sorry, I wrote one thing thinking about the other....